### PR TITLE
Moving info on China to our site.

### PR DESCRIPTION
**Pull request by sfshaza2**
Moving the developer info about China from the wiki to flutter.io.

### CODE REVIEW

File: src/community/china.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: Using Flutter in China
+description: Where to a version of flutter.io localized to Mandarin.
```
Comment (chalin) [inline src/community/china.md]: This description needs copy editing.

Comment (sfshaza2) [reply src/community/china.md]: Ee gads.

File: src/community/china.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+toc: false
+---
+
+The Flutter community has made a Mandarin version of the flutter.io
+website available at [https://flutter-io.cn](https://flutter-io.cn.).
```
Comment (chalin) [inline src/community/china.md]: - [ ] There is an **error in the link**: note the extraneous period at the end of the URL.
- [ ] IMHO we should drop the `https://` from the link text.

Comment (sfshaza2) [reply src/community/china.md]: I do now.  ;)

**Comment by sfshaza2** [comment]
Approved offline. Gale filed a bug against the broken GDG link (which I deleted): https://github.com/flutter/flutter/issues/23754

**Pull request merged**